deploy/template.yaml

```diff
@@ -1673,93 +1670,6 @@ Resources:
       FilterPattern: ""
       LogGroupName: !Ref CheckExistingIdentityFunctionLogGroup
 
-
-  ResetIdentityFunction:
```

Comment:
Do you think there's any risk of this breaking active sessions? I'm wondering if it might be safer to split this into two PRs:
- First add the new lambda and make all the journey map changes
- Second delete the old lambda

cc @Wynndow

Reply:
Brought back the `resetIdentity` lambda as part of this PR. Would create a second PR for the clean up after this is merged

Reply:
Thanks!